content-services/latest/develop/extension-inspector.md

```diff
@@ -103,25 +107,25 @@ When you run the Inventory command, the output is a report in json format with t
 }
 ```
 
-## Running the Analyser application
+## Run Analyser application
 
-The application analyses custom extensions against war inventories.
+The application analyzes custom extensions against WAR inventories.
 
-To run the Analyser application use the following command:
+Use the following command to run the application:
 
 ```java
-java -jar alfresco-extension-inspector-<version>.jar <extension-filename> [--
-target-version=6.1.0[-7.0.0] | --target-inventory =/path/to/war_inventory.json]
-[--verbose=[true | false]]
+java -jar alfresco-extension-inspector-<version>.jar <extension-filename>
+[--target-version=6.1.0[-7.0.0] | --target-inventory=/path/to/war_inventory.json]
+[--verbose=[true|false]]
 ```
 
-The following table lists the options for the application.
+where:
 
-|Option|Description|
-|------|-----------|
-|`--target-version`|An Alfresco version or a range of Alfresco versions|
-|`--target-inventory`|A file path of an existing WAR inventory|
-|`--verbose`|Verbose output|
+| Option | Description |
+| ------ | ----------- |
+| <nobr>--target-version</nobr> | *Required.* A specific Content Services version or range of versions. |
+| <nobr>--target-inventory</nobr> | *Required.* The file path to an existing WAR inventory. |
```

Comment:
`--target-version` and `--target-inventory` are **optional** and mutual exclusive.

Reply:
Added the new bits, thanks.